vm/devices/vmbus/vmbus_relay_intercept_device/src/lib.rs

```diff
@@ -438,17 +445,13 @@ impl<T: SimpleVmbusClientDeviceAsync> SimpleVmbusClientDeviceTask<T> {
         // one for the control bytes and at least one for the ring.
         assert!(page_count >= 4);
 
-        let shared_mem_spawner = LockedMemorySpawner;
-        let vmbus_mem = shared_mem_spawner
+        let mem = LowerVtlMemorySpawner::new(LockedMemorySpawner, self.vtl_protect.clone())
```

Comment:
This is OK but I'd rather we just pass this spawner thing in via trait object from underhill_core. Is that possible?

Reply:
Let me see... It should be possible, since that's what I do for the GET.

Reply:
Once you have that, it might be worth asking if we want to just move this into `underhill_core`, or at least move the crate under `openhcl/` instead of under `vm/devices/support`.

Reply:
Yep, let me move it too.